MobileFilter/sections/specific_web.txt

```diff
@@ -1155,6 +1155,7 @@ sibdom.ru##.fixed-position
 profile.ru###test_mob_ad
 ||hentaidude.com/wp-content/themes/awp/assets/mobile.js
 www-tellows-de.cdn.ampproject.org###singlecomments > li[style="padding:0px 5px 0px 5px;"]
+!+ NOT_OPTIMIZED
```

Comment:
Please move the rule and hint to the bottom where the other NOT_OPTMIZED rules are.

Move the two rules you mentioned in your comment to Mobile filter, name clearly indicates mobile, dunno why they are in normal filter.